sass/_themes.scss

```diff
@@ -1,8 +1,8 @@
 /* stylelint-disable color-no-hex */
 /* stylelint-disable value-keyword-case */
 // Colours based on Zeplin style guide 28march 2019
-$COLOR_WHITE        : #ffffff;
-$COLOR_BLACK        : #000000;
+$color-white        : #ffffff;
```

Comment:
I think the variable names should be all in upper-case, and when you access it in the styles below, you can use either upper or lower case depending on whatever is used around those lines to make it look consistent. Let's just do it this way and we can create a seperate card to make it all consistent across the whole file later on.

Reply:
@akmal-binary but as I was left with a comment Matin needs these constants to be in lower case, as well as the rest of the constants from the palette. I first changed these constants to upper case, then changed them to lower case, and now you recommend that I change them back to upper case. I little bit confuse...